lib/components/chat_history/twitch/prediction_event.dart

```diff
@@ -36,11 +36,9 @@ class TwitchPredictionEventWidget extends StatelessWidget {
 
 class _TwitchOutcomeWidget extends StatelessWidget {
   const _TwitchOutcomeWidget(
-      {Key? key,
-      required this.outcome,
+      {required this.outcome,
```

Comment:
Missing `super.key`

Reply:
A value for optional parameter 'key' isn't ever given.
Try removing the unused parameter.